samples/peripheral/802154_phy_test/README.rst

```diff
@@ -0,0 +1,356 @@
+     - custom lsetgpio 29 0
+   * - Get GPIO pin value
+     - custom lgetgpio <pin>
+     - <pin> - GPIO pin number, 0-N, where N depends on used SoC
+     - CMD will reconfigure selected GPIO pin to INPUT mode and read its value
+     - custom lgetgpio 29
+   * - Set DC/DC mode
+     - custom lsetdcdc <value>
+     - <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable DC/DC mode (no effect on unsupported boards)
+     - custom lsetdcdc 1
+   * - Get DC/DC mode
+     - custom lgetdcdc
+     - 
+     - Gets the DC/DC mode of CMD device, always 0 for unsupported boards
+     - custom lgetdcdc
+   * - Set ICACHE configuration
+     - custom lseticache <value>
+     - <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable ICACHE
+     - custom lseticache 1
+   * - Read SoC temperature
+     - custom lgettemp
+     - 
+     - CMD will print SoC temperature in the following format: <.%02>
+     - custom lgettemp
+   * - Transition radio to sleep mode
+     - custom lsleep
+     - 
+     - CMD will put radio into sleep mode
+     - custom lsleep
+   * - Transition radio to receive mode
+     - custom lreceive
+     - 
+     - CMD will put radio in receive mode
+     - custom lreceive
+   * - Reboot the device
+     - custom lreboot
+     - 
+     - Reboots the device
+     - custom lreboot
+
+
+   
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/peripheral/802154_phy_test`
+
+.. include:: /includes/build_and_run.txt
+
+.. note::
+   On the |nRF5340DKnoref|, the IEEE 802.15.4 PHY Test Tool is a standalone network sample that does not require any counterpart application sample.
+   However, you must still program the application core to boot up the network core (and forward UART pins to the network core of the CMD device).
+   The :ref:`nrf5340_empty_app_core`, which accomplishes both, is built and programmed automatically by default.
+   If you want to program another sample for the application core, unset the :option:'CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE' option.
+
+.. _802154_phy_test_testing:
+
+Using the tool
+==============
+
+After programming the supported development kit of your choice with the sample, you can verify that it functions as expected and begin performing tests.
+
+.. _802154_phy_test_testing_check:
+
+Checking if the sample works as expected
+----------------------------------------
+
+1. Connect the development kit to the computer using a USB cable. Use the DK's programmer USB port (J2).
+   The kits are assigned a COM port (Windows) or ttyACM device (Linux), which should be now visible in the Device Manager or /dev directory.
+#. Connect to the DK using a terminal emulator (for example, PuTTY or minicom), with the following settings:
+
+    - Baudrate: 115200
+    - Parity bits: none
+    - Stop bits: 1
+
+#. If the sample is configured to support BOTH modes (the default setting), switch the DK into CMD mode by sending the following command:
```

Comment:
This is a good place to mention the peculiarities of Phy Test Tool serial line (timeouts, no echo).